plugins/org.jkiss.dbeaver.data.gis/src/org/jkiss/dbeaver/data/gis/handlers/WKGUtils.java

```diff
@@ -84,7 +84,8 @@ public static boolean isCurve(@Nullable Object value) {
 
     @NotNull
     public static Object linearize(@NotNull Geometry value) {
-        return linearize(value, 1 / 32.0);
+        // This value results in 32 segments per quadrant, the default tolerance for ST_CurveToLine
+        return linearize(value, 0.001);
```

Comment:
1/32.0 is not equals to 0.001. Is it a mistake?

Reply:
It's not. `1/32` was my crude assumption on how the number of segments is computed, and it was wrong.